easybuild/easyconfigs/l/libgd/libgd-2.2.4-foss-2016b.eb

```diff
@@ -0,0 +1,30 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+
+easyblock = 'ConfigureMake'
+
+name = 'libgd'
+version = '2.2.4'
+
+homepage = 'http://libgd.bitbucket.org/'
```

Comment:
existing `libgd` easyconfig for 2.2.4 points to https://libgd.github.io, let's keep that consistent?